src/unistd.rs

```diff
@@ -819,6 +821,431 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
+          target_os="openbsd"))]
+SYMLOOP_MAX = libc::_SC_SYMLOOP_MAX,
+TIMER_MAX = libc::_SC_TIMER_MAX,
+TTY_NAME_MAX = libc::_SC_TTY_NAME_MAX,
+TZNAME_MAX = libc::_SC_TZNAME_MAX,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_CRYPT = libc::_SC_XOPEN_CRYPT,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_ENH_I18N = libc::_SC_XOPEN_ENH_I18N,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_LEGACY = libc::_SC_XOPEN_LEGACY,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_REALTIME = libc::_SC_XOPEN_REALTIME,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_REALTIME_THREADS = libc::_SC_XOPEN_REALTIME_THREADS,
+_XOPEN_SHM = libc::_SC_XOPEN_SHM,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="openbsd"))]
+_XOPEN_STREAMS = libc::_SC_XOPEN_STREAMS,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_UNIX = libc::_SC_XOPEN_UNIX,
+#[cfg(any(target_os="android", target_os="dragonfly", target_os="freebsd",
+          target_os = "ios", target_os="linux", target_os = "macos",
+          target_os="openbsd"))]
+_XOPEN_VERSION = libc::_SC_XOPEN_VERSION,
+}
+
+/// Get configurable system variables
+///
+/// Returns the value of a configurable system variable.  Most supported
+/// variables also have associated compile-time constants, but POSIX
+/// allows their values to change at runtime.  There are generally two types of
+/// sysconf variables: options and limits.  See sysconf(3) for more details.
+///
+/// # Returns
+///
+/// - Ok(Some(x)): the variable's limit (for limit variables) or its
+///     implementation level (for option variables).  Implementation levels are
+///     usually a decimal-coded date, such as 200112 for POSIX 2001.12
+/// - Ok(None): the variable has no limit (for limit variables) or is
+///     unsupported (for option variables)
+/// - Err(x): an error occurred
+///
+/// http://pubs.opengroup.org/onlinepubs/9699919799/functions/sysconf.html
+pub fn sysconf(var: SysconfVar) -> Result<Option<c_long>> {
+    let raw = unsafe {
+        Errno::clear();
+        libc::sysconf(var as c_int)
+    };
+    if raw == -1 {
```

Comment:
No, because `sysconf` will return -1 with `errno` set to 0 to indicate that the requested variable is not supported.  Our `Errno::result` class can't handle that.